packages/cozy-client/src/store/queries.js

```diff
@@ -48,6 +48,7 @@ const queryInitialState = {
   id: null,
   definition: null,
   fetchStatus: 'pending',
+  isFetching: false,
```

Comment:
I am afraid this implementation will affect Node part. But I wonder how to add the feature elsewhere. 🤔

Reply:
Maybe my implementation should stay closer to fetchPolicy. But in my opinion, add a queryResult here, is practical for usage. (see in the examples of react-integration.md)

Reply:
It'll affect since we'll have a new QueryState attribute, but it's fine.